appveyor-clean-cache.txt

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Do we need to check this file into the repository? Should we remove it here and add it to `.gitignore` instead?